src/app.js

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+
+const searchForm = document.getElementById("search-form");
+searchForm.addEventListener("submit",handleForm);
+
+const errorInfo = document.getElementById("error");
+const countryInfo = document.getElementById("country-info");
+
+function handleForm(e) {
+    e.preventDefault();
+
+    const inputField = document.getElementById("input-field");
+    getCountryData(inputField.value);
+
+    inputField.value = "";
+}
+
+async function getCountryData(name) {
+    countryInfo.innerHTML = ``;
+    errorInfo.innerHTML = ``;
+
+    try{
+        const result = await axios.get(`https://restcountries.com/v2/name/${name}`);
+        const country = result.data[0];
```

Comment:
Je zou hier ook het volgende van kunnen maken
```suggestion
        const { flag, name, population, region, capital, currencies, languages } = result.data[0];
```
Dan hoef je vervolgens nergers meer `country.` vóór te zetten